pkg/reloader/example_test.go

```diff
@@ -49,5 +49,5 @@ func ExampleReloadURLFromBase() {
 		log.Fatal(err)
 	}
 	fmt.Println(reloader.ReloadURLFromBase(u))
-	// Output: http://localhost:9090/-/reload
+	//    Output: http://localhost:9090/-/reload
```

Comment:
This is the only exception where we used `// <multiple spaces>` I think we can add trailing period, rem spaces and remove supporting this exception.. What do you think?

Reply:
I think some lists uses it also such as
```
// foo:
//   - bar
//   - bar
...
```

Not sure if lists should also end with period?